### ISSUE TITLE
java.lang.NullPointerException for xml dumps

### ISSUE THREAD

**Issue by alismayilov** [Wikidata/Wikidata-Toolkit#301]
We got a lot of NullPointerException when we use wdtk 0.7.0 with latest xml dumps. I thought I can catch those exceptions, but a number of exceptions are more than normal. We got exceptions when we get values (from value.toString, getValue, etc). We did not have those exceptions before. Could you please verify this?

**Comment by Tpt** [comment]
Thank you for the bug report. In order to don't waste time looking in wrong directions could you report on which items and values these NullPointerException are raised? If they are related to quantity values it is solved in the master branch.

**Comment by alismayilov** [comment]
Thanks for the quick response! I verified for the small dump file. It seems all of them are from JacksonValueQuantity. When are you planning to release this fix? I mean we use maven, it seems 0.8.0 is not available in maven.

**Comment by sstdz** [comment]
I think that is because of using toString() methods. They are overriden for some childs of Value class and use the help of additional class ToString. For example, for QuantityValue it has such view:
```
public static String toString(QuantityValue o) {
		return o.getNumericValue().toString() + " ["
				+ o.getLowerBound().toString() + " .. "
				+ o.getUpperBound().toString() + "]" + " " + o.getUnit();
	}
```
And often, in this option, when you grab the data, the fields `lowerBound` and `upperBound` are null, so this is where it taken.

**Comment by Tpt** [comment]
Thank you for the investigation. This problem is fixed in the master branch

**Comment by sstdz** [comment]
Thank you :)

**Comment by Tpt** [comment]
It should be fixed in the new 0.8.0 version
